docs/source/_src_user_guide/state_propagation/propagation_setup/dynamics_types/multi_arc.rst

```diff
@@ -0,0 +1,33 @@
+.. _multi_arc_dynamics:
```

Comment:
Is it difficult to change multi_arc.rst into multi_hybrid_arc.rst, for the same reason mentioned previously?

Reply:
Nope, fixed